CRM/Certificate/BAO/CompuCertificateImageFormat.php

```diff
@@ -0,0 +1,267 @@
+<?php
+
+class CRM_Certificate_BAO_CompuCertificateImageFormat extends CRM_Core_DAO_OptionValue {
```

Comment:
I actually implemented this as a standalone entity before but later changed it, so that it can be consistent with the implementation of CRM_Core_BAO_PdfFormat

Also, in a later PR, the ImageFormat will be placed inside the message template page and used the same way the pdf format is used.
![image](https://user-images.githubusercontent.com/85277674/212086326-7e22f33a-6f48-4b2e-83b2-cf85f6ee4023.png)

3rd reason is that using the option group allows for flexible image format fields, at least on the DB side.

Reply:
Then, for the first concern, if the ImageFormat is needed. I already raised it with Jamie and Pallavi, and we agree it's needed.

Reply:
>  and we agree it's needed.

I don't think saying that it is "needed" without mentioning 'why" is a good enough a reason, my reason on why it is not needed is that there are not that much of formats in which people export certificates on, they are either a PDF or an image, and if it is an image then in most cases it is either "PNG" or "JPEG", that is why I don't see the point in having it configurable, it basically adds an unnecessary complication. So again I would like to see a good "reason" on why they think it is needed.

Reply:
> if we don’t have the ability to define output formats how does the system know how the output should look in terms of the size and quality of PNG/JPEG?


with the Image format field, we can define the quality, width and height of the image.

though we can still argue that these can be determined by the HTML code in the message template. except for extension (JPG/PNG).